lms/product/blackboard/_plugin/course_copy.py

```diff
@@ -0,0 +1,25 @@
+from lms.product.plugin.course_copy import CourseCopyPlugin
+
+
+class BlackboardCourseCopyPlugin(CourseCopyPlugin):
```

Comment:
Blackboard plugin just override the BB specific bits, getting the files from the API.